src/Network/Probe.php

```diff
@@ -1706,7 +1706,7 @@ private static function twitter($uri)
 	{
 		if (preg_match('=(.*)@twitter.com=i', $uri, $matches)) {
 			$nick = $matches[1];
-		} elseif (preg_match('=https?://twitter.com/(.*)=i', $uri, $matches)) {
+		} elseif (preg_match('=https?://(?:mobile.)?twitter.com/(.*)=i', $uri, $matches)) {
```

Comment:
The first two dots need to be escaped in this regular expression. Otherwise it would match `mobilestwitter.com` as well.

Reply:
i only introduced one additional dot, which other dot do you mean? the one in twitter.com?

Reply:
Yes. It is less a problem because there's no chance of a "twitterocom" TLD, but it still is bad practice.